cdoc/cdoc_jar/src/main/java/org/cdoc4j/cli/Tool.java

```diff
@@ -0,0 +1,164 @@
+package org.cdoc4j.cli;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.GeneralSecurityException;
+import java.security.cert.X509Certificate;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+
+import org.cdoc4j.CDOC;
+import org.cdoc4j.CDOCBuilder;
+import org.esteid.sk.IDCode;
+import org.esteid.sk.LDAP;
+
+import static spark.Spark.get;
+import static spark.Spark.port;
+import static spark.Spark.post;
+
+
+
+
+public class Tool {
+
+
+    public static void main(String[] args) {
+        port(4444);
+
+        post("/cdoc", (request, response) -> {
+            String jarArgs = request.body();
+            String[] argParts = jarArgs.split("&");
+            String nameFrom = argParts[0];
+            String nameTo = argParts[1];
+            String idCode = argParts[2];
+            
+            String fileName = nameFrom+".txt";
+
+            try {
+                File plain = new File(fileName);
```

Comment:
From a security perspective, where is this file written to? There are usually functions to determine the OS tmp folder, bad practice to write to the same folder where the application resides.

Reply:
This doesn't seem like an issue, since from a security perspective, the data is POST-ed in a decrypted form anyways between services.